src/shared/SellerProfile/PrivateInfo.js

```diff
@@ -113,6 +115,23 @@ const PrivateInfo = (props) => {
                     </p>
                 }
             </Row>
+            <Row title="Labour hire licence" show={!isEmpty(labourHire)}>
+              {Object.keys(labourHire).map((key, i) => {
+                return (
+                  (labourHire[key]['licenceNumber'] || labourHire[key]['expiry']) && (
+                    <div key={i}>
```

Comment:
Not an issue for me but the block in Body.js used a React fragment.  Just raising this in case you want them to be consistent.

Reply:
Probably...but as this has passed testing now, reluctant to change in case it has any styling implications